titanic.ipynb

```diff
@@ -5,14 +5,16 @@
       "cell_type": "markdown",
```

Comment:
_Commented on notebook `titanic.ipynb` Cell 1 Line 5_
```markdown
## Titanic Dataset Analysis

<img src="https://www.pigeonforgetncabins.com/wp-content/uploads/2015/06/experience-at-titanic-pigeon-forge.jpg" width="300" />

Using pandas to analyze the the [titanic dataset](https://github.com/mwaskom/seaborn-data/blob/master/titanic.csv) to showcase GitNotebooks
```
what

_Respond and view the context [here](http://localhost:3000/smith-kyle/notebook-diff-tests/pull/25)._